pkg/util/baseyamlutil.go

```diff
@@ -47,30 +47,25 @@ func ConvertYamlFileToRuntimeObjects(stringContent string) map[string]runtime.Ob
 
 	for _, singleYaml := range listOfSingleK8sResourceYaml {
 		if singleYaml == "\n" || singleYaml == "" {
-			// ignore empty cases
-			//log.V(1).Info("Got empty", "here", singleYaml)
 			continue
 		}
 
 		decode := scheme.Codecs.UniversalDeserializer().Decode
 		runtimeObject, groupVersionKind, err := decode([]byte(singleYaml), nil, nil)
 		if err != nil {
-			//log.V(1).Info("unable to decode a single yaml object, skipping", "singleYaml", singleYaml, "error", err)
-			continue
+			return nil, err
```

Comment:
Why does this check return an error but the checks before and after this one do `continue`?

Reply:
I get that this check is a little more important than the next one :) but maybe the next one should also error or log an Error "Unable to create ID for 'kind' due to 'err'"

Reply:
Fixed. We'll just return the error.